src/db/oracledb/persons-dao.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+import { serializePerson } from '../../serializers/persons-serializer';
+
+/**
+ * Queries data source for raw person data and passes it to the serializer
+ *
+ * @param {string} osuId 9-digit osu ID of a person
+ * @returns {object} Serialized person resource from person-serializer
+ */
+const getPersonById = async (osuId) => {
+  const connection = await getConnection();
+  try {
+    const { rows } = await connection.execute(contrib.getPersonById(osuId));
```

Comment:
I don't think `osuId` should be parameter for `getPersonById`. It should be passed as a binding variable. See my comment in contirb repo.